wandb/sdk/wandb_run.py

```diff
@@ -2499,6 +2506,9 @@ def _log_artifact(
         is_user_created: bool = False,
         use_after_commit: bool = False,
     ) -> wandb_artifacts.Artifact:
+        api = internal.Api()
+        if api.settings().get("anonymous") == "true":
+            wandb.termwarn("Artifacts won't be claimed when you claim the run.")
```

Comment:
Maybe reword to: `Artifacts logged anonymously cannot be claimed and expire after 7 days.`